docs/database/ef-core-migrations.md

```diff
@@ -73,9 +89,63 @@ Start by creating some migrations to apply.
 
```

Comment:
If we're adding the package manager console for the previous migration, we should add it here too. Again, tabs would be good instead of H3 sections.